src/components/Homepage.js

```diff
@@ -1,70 +1,161 @@
+/* eslint-disable react/no-array-index-key */
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import Faq from 'react-faq-component';
-import { FaqContent, WelcomeText } from './Text';
+import { FaqContent } from './Text';
 import './HomepageStyles.scss';
+import smile from '../img/smile.svg';
+import heart from '../img/heart.svg';
+import rightArrow from '../img/rightArrow.png';
+
+class Application extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      welcomes: [
+        { content: <p className="f61"> A tool to <i>capture</i> the present, <i>remember</i> the past, and <i>refresh</i> your outlook on the future</p> },
+        { content: <p className="f36">But how? In order to prevent us from hopping on the next quick thing, for future reference not only captures but is also a reminder of the current circumstances in order to strengthen us and one another. Following a time of reflection we want to give the opportunity to give encouragement and thanks to the people who’ve helped us along the way.</p> },
+      ],
+      itemsToShow: 1,
+      expanded: false,
+    };
+
+    this.showMore = this.showMore.bind(this);
+  }
+
+  /* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
+  showMore() {
+    this.state.itemsToShow === 1 ? (
+      this.setState({ itemsToShow: 2, expanded: true })
+    ) : (
+      this.setState({ itemsToShow: 1, expanded: false })
+    );
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className="row">
+          <ul>
```

Comment:
If you center these and make the outer class a flexbox, it should resize with the page!